EasyReflectometry/sample/elements/layers/layer_apm.py

```diff
@@ -22,7 +22,7 @@
         'max': np.inf,
         'fixed': True,
     },
-    'chemical_structure': 'C10H18NO8P',
+    'molecular_formula': 'C10H18NO8P',
```

Comment:
Not to sound too nitpicky, but will we only have molecules here? No single atoms allowed? Or other non-molecules?
Maybe `chemical_formula` would sound more appropriate?
Just a question.

Reply:
changed to `chemical_formula`